lib/cmd/fh3/undeploy.js

```diff
@@ -0,0 +1,50 @@
+/* globals i18n */
+module.exports = undeploy;
+undeploy.undeploy = undeploy;
```

Comment:
This is the way that is all commands. So, I believe that is not good we do differently just in one. If it will be changed we can do it in all horizontally.